app/views/user_mailer/admin_deleted_work_notification.html.erb

```diff
@@ -1,11 +1,11 @@
 <% content_for :message do %>
-  <p>Dear <%= @user.default_pseud.byline %>,</p>
+  <p><%= t '.hello', user: @user.default_pseud.byline %></p>
   
-  <p>Your work <i><%= style_bold(@work.title.html_safe) %></i> was deleted from the Archive by a site admin.</p>
+  <p><%= t('.html.part1', title: style_bold(@work.title.html_safe)).html_safe %></p>
```

Comment:
Actually, it looks like we did `<i>#{style_bold(@work.title.html_safe)}</i>".html_safe` in #2121 rather than put the italics in the .yml files: https://github.com/otwcode/otwarchive/pull/2121/files#diff-f84523f05ea8469c1afdf9debe591f4dR2